### DISCUSSION TITLE
Cannot use payload.find on the client side.

### DISCUSSION THREAD

**Discussion by jamal7887** [payloadcms/payload#3414] [Q&A]
### Link to reproduction

Not relevant

### To Reproduce

Use the payload.find method on the client side

### Describe the Bug

I'm facing an error when trying to use payload.find on the client side. I'm curious about why this method isn't supported on the client side.

### Payload Version

1.15.6

**Comment by PatrikKozak** [comment]
Hey @jamal7887 - The `payload.find` method and similar methods from Payload's [local API](https://payloadcms.com/docs/local-api/overview) are designed to be executed within Node, directly on your server.

If you're looking to fetch or manipulate data on the client side using Payload, the recommended approach is to utilize Payload's [REST](https://payloadcms.com/docs/rest-api/overview) or [GraphQL](https://payloadcms.com/docs/graphql/overview) APIs.

If you have any other questions, please let me know. I'm here to help.

Going to convert this to a discussion.
